data-prepper-plugins/peer-forwarder/src/test/java/com/amazon/dataprepper/plugins/processor/peerforwarder/discovery/DiscoveryMode.java

```diff
@@ -0,0 +1,7 @@
+package com.amazon.dataprepper.plugins.processor.peerforwarder.discovery;
+
+public enum DiscoveryMode {
+    STATIC,
+    DNS,
+    UNSUPPORTED
```

Comment:
QUES: do we need this UNSUPPORTED option? When will it be selected?

Reply:
I'll leave a comment in the file. 

This is actually an enum defined in the test/ directory solely for testing a branch in the [PeerListProviderFactory](https://github.com/opendistro-for-elasticsearch/Data-Prepper/blob/master/data-prepper-plugins/peer-forwarder/src/main/java/com/amazon/dataprepper/plugins/processor/peerforwarder/discovery/PeerListProviderFactory.java#L42) switch statement. It feels somewhat hacky to define an enum just for testing, but the alternative was to leave it untested.

Reply:
Ah. Sorry, did not notice that it is in the test.